### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by farishkash** [mbsk3lton/prj-rev-bwfs-tea-cozy#4]
This project needs improvement.  The website looked like the mock except for the problem with the whole website being on the left side of the screen.  I had added margin 0 auto to the major divs and that start to pull things in the middle.

The reason why I feel like this needs a major rewrite is you have a lot of unneeded css here. Either get rid of it of the extra css or make this site work with flexbox.  Will let you know that once you start adding display:flex everything is going to shift around dramatically.

Leading to that, I am going to reiterate that `display:flex `needs to be put into the css for each div  otherwise it doesn't work.

If you are struggling with flexbox here are a couple games to play that may help you.

Flexboxdefense.com and flexboxfroggy.com

Flexboxdefense isn't as advance so I would do that first then try flexboxfroggy.com and see if you can clear all 24 stages.

**Comment by mbsk3lton** [comment]
Yes this was very difficult and challenging. I will rewrite this and should I contact you for a reevalation?
